test/integration/targets/apt/tasks/apt.yml

```diff
@@ -179,6 +179,35 @@
 - name: uninstall hello with apt
   apt: pkg=hello state=absent purge=yes
 
+# INSTALL WITHOUT REMOVALS
+- name: Install hello, that conflicts with hello-traditional
+  apt: pkg=hello state=present update_cache=no
+
+- name: check hello
+  shell: dpkg-query -l hello
+  failed_when: False
```

Comment:
Why is failure being ignore here?

Reply:
I verified the installation of "hello" package in the same manner as the existing test.

Please see here:
https://github.com/ansible/ansible/blob/308723c3ca1122363e419070f1fa1d76ff5611a9/test/integration/targets/apt/tasks/apt.yml#L116-L130

This task should success, so it is not required to ignore a failure as you say.
Should I remove this line from my PR?

Reply:
Yes. Tests should not ignore failure unless absolutely necessary.